zig/examples/basic.zig

```diff
@@ -0,0 +1,42 @@
+const std = @import("std");
+const testing = std.testing;
+const sdk = @import("extism-sdk");
```

Comment:
I will defer to your Zig expertise here, but conventionally we are naming the SDK packages `extism`, and then to refer to the top-level namespace as `extism`. So:

```zig
const extism = @import("extism");
const Context = extism.Context;
const Plugin = extism.Plugin;
// ... and so on
```

If there are idioms in the Zig ecosystem that would suggest otherwise, I think we should change this to match other SDKs.

Reply:
I named it without looking other libraries, will push a commit to rename to extism